Misc/NEWS.d/next/Library/2020-05-23-18-41-53.bpo-40744.q_4vJB.rst

```diff
@@ -0,0 +1 @@
+Drop support for sqlite3 pre v3.7.3
```

Comment:
Nitpick: sqlite3 -> SQLite

Reply:
And please add "Patch by Erlend Egeberg Aasland.".

Reply:
> I just saw that you've combined changes from #17413. I think it would be better if you could add Sergey's email address in the co-authored-by trailer so he could properly attributed:
> 
> ```
> Sergey Fedoseev <fedoseev.sergey@gmail.com>
> ```
> 
> Please add Sergey's name to the NEWS entry as well.

Of course!